pytrello2/board.py

```diff
@@ -35,10 +35,17 @@ def get_all_boards(self):
 
     def create_board(self, name, desc):
         """
-        Returns a Card object with the given ID.
+        Returns a Board object with the given ID.
```

Comment:
Nice catch, but the doc string should explain what the function is about. Maybe this is not the correct explanation.

WDYT?

Reply:
Agreed, will change it.